### ISSUE TITLE
Cannot Find Module

### ISSUE THREAD

**Issue by jrock2004** [antonk52/cssmodules-language-server#8]
On my one machine with Neovim, I am getting lsp error and not sure what caused this.

[ERROR][2022-01-24 16:54:36] .../vim/lsp/rpc.lua:420	"rpc"	"cssmodules-language-server"	"stderr"	"internal/modules/cjs/loader.js:892\n  throw err;\n  ^\n\nError: Cannot find module './utils/resolveAliasedImport'\nRequire stack:\n- /Users/John.Costanzo/.local/share/nvim/lsp_servers/cssmodules_ls/node_modules/cssmodules-language-server/lib/utils.js\n- /Users/John.Costanzo/.local/share/nvim/lsp_servers/cssmodules_ls/node_modules/cssmodules-language-server/lib/CompletionProvider.js\n- /Users/John.Costanzo/.local/share/nvim/lsp_servers/cssmodules_ls/node_modules/cssmodules-language-server/lib/connection.js\n- /Users/John.Costanzo/.local/share/nvim/lsp_servers/cssmodules_ls/node_modules/cssmodules-language-server/lib/cli.js\n    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:889:15)\n    at Function.Module._load (internal/modules/cjs/loader.js:745:27)\n    at Module.require (internal/modules/cjs/loader.js:961:19)\n    at require (internal/modules/cjs/helpers.js:92:18)\n    at Object.<anonymous> (/Users/John.Costanzo/.local/share/nvim/lsp_servers/cssmodules_ls/node_modules/cssmodules-language-server/lib/utils.js:22:32)\n    at Module._compile (internal/modules/cjs/loader.js:1072:14)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)\n    at Module.load (internal/modules/cjs/loader.js:937:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:778:12)\n    at Module.require (internal/modules/cjs/loader.js:961:19) {\n  code: 'MODULE_NOT_FOUND',\n  requireStack: [\n    '/Users/John.Costanzo/.local/share/nvim/lsp_servers/cssmodules_ls/node_modules/cssmodules-language-server/lib/utils.js',\n    '/Users/John.Costanzo/.local/share/nvim/lsp_servers/cssmodules_ls/node_modules/cssmodules-language-server/lib/CompletionProvider.js',\n    '/Users/John.Costanzo/.local/share/nvim/lsp_servers/cssmodules_ls/node_modules/cssmodules-language-server/lib/connection.js',\n    '/Users/John.Costanzo/.local/share/nvim/lsp_servers/cssmodules_ls/node_modules/cssmodules-language-server/lib/cli.js'\n  ]\n}\n"

**Comment by MagicDuck** [comment]
same problem here. It's because when the npm package is packed up, it seems to miss `lib/utils/resolveAliasedImport.js` (compiled from the corresponding ts file). I am guessing it doesn't grab files recursively. 
Worked around it by rolling back to the older version:
```
yarn global remove cssmodules-language-server
yarn global add cssmodules-language-server@1.1.0
```

**Comment by MagicDuck** [comment]
Guessing this might need to change:
https://github.com/antonk52/cssmodules-language-server/blob/main/package.json#L24
Possibly to:
```
  "files": [
    "lib/**/*.js",
    "lib/**/*.d.ts"
  ],
  ```

**Comment by antonk52** [comment]
Thank you for the issue and looking into why this fails. I am going to release a new version later today. If you want to send a PR in the meanwhile I would be happy to merge it.

**Comment by MagicDuck** [comment]
Thanks @antonk52! I've created a PR addressing the issue: https://github.com/antonk52/cssmodules-language-server/pull/9

**Comment by antonk52** [comment]
Merged and released in [v1.2.1](https://github.com/antonk52/cssmodules-language-server/releases/tag/v1.2.1). Thank you!

**Comment by MagicDuck** [comment]
Thank you for creating & sharing this handy tool @antonk52! 😄
